### PR TITLE
Upgrade to go 1.13

### PR DESCRIPTION
**Pull request by chenrui333**
Good to keep it with the latest golang version. :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
 language: go
 
-sudo: false
```
Comment (chenrui333) [inline .travis.yml]: This is no longer needed for Travis linux infra.
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

**Comment by wata727** [comment]
Looks good, but Azure Pipelines still run with Go 1.12...
https://dev.azure.com/watassbass/watassbass/_build/results?buildId=257

**Comment by wata727** [comment]
Maybe the following documentation helps us.
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/go-tool?view=azure-devops

**Comment by wata727** [comment]
The Azure Pipelines Go version has been updated in #433. I will merge this one. Thank you!

**Comment by chenrui333** [comment]
Thanks for taking care of this PR.

**Pull request merged**